python/external_task_api_client/external_task_api_client_service.py

```diff
@@ -0,0 +1,71 @@
+import aiohttp
+import asyncio
+import json
+
+
+class ExternalTaskApiClientService:
+
+    def __init__(self, baseUrl):
+        self.__baseUrl = baseUrl
+
+    async def extendLock(self, identity, workerId, externalTaskId, additionalDuration):
+        uri = "task/" + externalTaskId + "/extend_lock"
+        request = {
+            "workerId": workerId,
+            "additionalDuration": additionalDuration
+        }
+
+        return await self.__sendPostToExternalTaskApi(identity, uri, request)
+
+    async def fetchAndLockExternalTasks(self, identity, workerId, topicName, maxTasks, longPollingTimeout, lockDuration):
```

Comment:
```suggestion
    async def fetch_and_lock_external_tasks(self, identity, worker_id, topic, max_tasks, long_polling_timeout, lock_duration):
```

Ich würde `topicName` zu `topic` ändern, da es anderswo nur als topic auftaucht.